substrate/bin/node/cli/src/chain_spec.rs

```diff
@@ -296,7 +298,7 @@ fn configure_accounts(
 }
 
 /// Helper function to create RuntimeGenesisConfig json patch for testing.
-pub fn testnet_genesis(
+pub fn testnet_genesis_patch(
```

Comment:
Hm, could we just move all this genesis-related code to [`kitchensink::genesis_config_presets`](https://github.com/paritytech/polkadot-sdk/blob/aa67e2dbda7d68238b62325dec791af157c7ccaf/substrate/bin/node/runtime/src/genesis_config_presets.rs), and just provide regular named presets?

I think that would be cleaner and there would be no duplication.
Not sure however if all scenarios will be covered.

Note: could be done in another follow-up.

Reply:
In this particular case this is not possible unfortunately unless the mechanism is changed. The account setup here is dynamically generated based on `std::random` and environment variables to create different setups for testing staking related code.

@kianenigma is the owner in this aspect. Do you still fancy the dynamic setup? :D